frontend/src/components/ProductCard/types.ts

```diff
@@ -1,8 +1,9 @@
 export interface ProductCardProps {
   id: number;
   title: string;
-  price: number;
+  price?: number;
   size: "small" | "medium" | "large";
   priceBeforeSale?: number;
   backgroundImage: string;
+  isPreview: boolean;
```

Comment:
Consider making isPreview an optional field and setting its default value to false in the ProductCard component. This way, you can omit the prop when false, and the component will automatically treat it as false by default. Only pass the prop when true is needed.